js/project-card.js

```diff
@@ -6,6 +6,8 @@ var ProjectCard = {
     var id = projectData.id;
     var timestamp = new Date(projectData.Timestamp);
     var month = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"][timestamp.getMonth()];
+    var issues = projectData.Issues ? projectData.Issues : "";
```

Comment:
For the next iteration, if it's still a spreadsheet, I think we should consider making each issue its own column with a boolean value. But kudos to all this array handling! 👏 👏